### ISSUE TITLE
tracking additional files with out of date git-lfs causing failures on travis-ci

### ISSUE THREAD

**Issue by lbergelson** [github/git-lfs#683]
We've been using git-lfs to manage several large files with some success so far.  

However we seem to have run into a problem that I don't know how to deal with.

As best as I understand this is what has happened:
one of our team members who was using git-lfs v0.5.4 tracked and committed a new file.  He tried to push it but it didn't work, at which point he upgraded to v0.6.0 which seemed to work.  His build passed on travis and he merged it.  However, the merge build is failing because the git-lfs managed files are not being downloaded now.

Here's a failing travis log.  The relevant lines are:

(pull is echoed right before we run git lfs pull)
```
pull

(0 of 10 files) 0 B / 412.21 MB                                                
Invalid status for POST https://github.com/broadinstitute/hellbender.git/info/lfs/objects/batch: 403
```

[lfs-failing-log.txt | uploaded via ZenHub](https://files.zenhub.io/5605aa16c4c3126521f44b9b)
We have a test that detects if the files are properly checked out, and it is failing.  The files are not being replaced with the large version.

However, a local clone of our project succeeds at downloading the files. 
If you do an explicit pull though, you get 

```
hellbender git:(master) git lfs pull
Problem accessing src/test/resources/large/exampleLargeFile.txt

Errors logged to /Users/louisb/tmp/hellbender/.git/lfs/objects/logs/20150925T161320.682693917.log
Use `git lfs logs last` to view the log.
(0 of 10 files) 0 B / 412.21 MB
```

the log output is 
```
git-lfs/0.6.0 (GitHub; darwin amd64; go 1.5.1)
git version 2.5.3

$ git-lfs pull
Problem accessing src/test/resources/large/exampleLargeFile.txt

Expected key version, got This
goroutine 83 [running]:
github.com/github/git-lfs/lfs.Stack(0x0, 0x0, 0x0)
	/private/tmp/git-lfs20150911-41373-1j122xr/git-lfs-0.6.0/src/github.com/github/git-lfs/lfs/errors.go:525 +0x80
github.com/github/git-lfs/commands.logPanicToWriter(0xaa41c0, 0xc820076040, 0xaa0000, 0xc8200d2080)
	/private/tmp/git-lfs20150911-41373-1j122xr/git-lfs-0.6.0/src/github.com/github/git-lfs/commands/commands.go:181 +0xf7f
github.com/github/git-lfs/commands.logPanic(0xaa0000, 0xc8200d2080, 0x0, 0x0)
	/private/tmp/git-lfs20150911-41373-1j122xr/git-lfs-0.6.0/src/github.com/github/git-lfs/commands/commands.go:145 +0x421
github.com/github/git-lfs/commands.handlePanic(0xaa0000, 0xc8200d2080, 0x0, 0x0)
	/private/tmp/git-lfs20150911-41373-1j122xr/git-lfs-0.6.0/src/github.com/github/git-lfs/commands/commands.go:120 +0x4e
github.com/github/git-lfs/commands.LoggedError(0xaa0000, 0xc8200d2080, 0x4c8330, 0x14, 0xc820061ee8, 0x1, 0x1)
	/private/tmp/git-lfs20150911-41373-1j122xr/git-lfs-0.6.0/src/github.com/github/git-lfs/commands/commands.go:70 +0x82
github.com/github/git-lfs/commands.checkoutWithChan(0xc820098e40)
	/private/tmp/git-lfs20150911-41373-1j122xr/git-lfs-0.6.0/src/github.com/github/git-lfs/commands/command_checkout.go:184 +0x2fd
github.com/github/git-lfs/commands.checkoutFromFetchChan.func1(0xc820098e40, 0xc820118030)
	/private/tmp/git-lfs20150911-41373-1j122xr/git-lfs-0.6.0/src/github.com/github/git-lfs/commands/command_checkout.go:79 +0x21
created by github.com/github/git-lfs/commands.checkoutFromFetchChan
	/private/tmp/git-lfs20150911-41373-1j122xr/git-lfs-0.6.0/src/github.com/github/git-lfs/commands/command_checkout.go:81 +0x439

ENV:
LocalWorkingDir=/Users/louisb/tmp/hellbender
LocalGitDir=/Users/louisb/tmp/hellbender/.git
LocalGitStorageDir=/Users/louisb/tmp/hellbender/.git
LocalMediaDir=/Users/louisb/tmp/hellbender/.git/lfs/objects
TempDir=/Users/louisb/tmp/hellbender/.git/lfs/tmp
ConcurrentTransfers=3
BatchTransfer=true
```

Our project is https://github.com/broadinstitute/hellbender 

We're running our tests on travis CI, and using git-lfs by downloading and building it on travis using this script: https://github.com/broadinstitute/hellbender/blob/master/scripts/install_git_lfs.sh

I don't understand what is failing or how to fix it.  It seems like a full checkout works if you have git lfs installed already, but running a pull fails.  Does anyone have suggestions?

**Comment by lbergelson** [comment]
Update:  re-running our travis job has fixed the test failure, but the command line errors remain.  It's unclear if this is a transient issue or a recurring one...

**Comment by technoweenie** [comment]
Your file is too small to be used with LFS :trollface: 

> Expected key version, got This

Since your file is below the blob threshold (which is 1024 bytes right now), it's still treated like an LFS pointer. It bombs because it's not. I should be able to tweak the function to let tiny test files like yours through.

**Comment by lbergelson** [comment]
Interesting.  Why did it start happening suddenly though?  It was fine before.

**Comment by technoweenie** [comment]
I'm not exactly sure, but there were some big changes to how Git LFS scans the repo for LFS objects between 0.5.4 and 0.6.0. This is a pretty minor regression IMO.

**Comment by technoweenie** [comment]
Should be fixed by #684.

**Comment by lbergelson** [comment]
this seems to be resolved, thank you!
